client/pom.xml

```diff
@@ -21,8 +21,6 @@
                 <configuration>
                     <instructions>
                         <Karaf-Commands>*</Karaf-Commands>
-                        <!-- Nothing to export -->
-                        <Export-Package></Export-Package>
```

Comment:
Should be `<Export-Package>org.opennms.velocloud.client.*</Export-Package>` or it will re-export all dependencies.